lib/mapboxkit/connection.rb

```diff
@@ -53,7 +53,7 @@ def connection
 
             faraday.response(:logger) if ENV['VERBOSE'] == '1'
 
-            faraday.response(:json)
+            faraday.response(:json, content_type: %r{^application/(vnd\..+\+)?json$})
```

Comment:
FYI: This regexp is from: https://github.com/lostisland/faraday/blob/main/lib/faraday/request/json.rb#L16